week-1/explore/objects.md

```diff
@@ -4,13 +4,19 @@ In this activity, we'll explore some additional concepts that you'll encounter i
 
 Open the Chrome devtools Console, type in `console.log` and then hit enter
 
-What output do you get?
+What output do you get? 
+ƒ log() { [native code] }
 
 Now enter just `console` in the Console, what output do you get back?
+console {debug: ƒ, error: ƒ, info: ƒ, log: ƒ, warn: ƒ, …}
 
 Try also entering `typeof console`
+'object'
 
 Answer the following questions:
 
-What does `console` store?
+What does `console` store? 
+The Console can be used to log information as part of the JavaScript development process, as well as allow you to interact with a web page by carrying out JavaScript expressions within the page's context.
```

Comment:
The question refers to the `console` javascript object, not the developer console. What does that object store?